SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -201,3 +172,108 @@ Worker_ComponentId USpatialDispatcher::GetComponentId(Worker_Op* Op) const
 		return SpatialConstants::INVALID_COMPONENT_ID;
 	}
 }
+
+USpatialDispatcher::CallbackId USpatialDispatcher::AddOpCallback(Worker_ComponentId ComponentId, const UserCallback<Worker_AddComponentOp>& Callback)
+{
+	return AddGenericOpCallback(ComponentId, WORKER_OP_TYPE_ADD_COMPONENT, [Callback](const Worker_Op* Op)
```

Comment:
Do we need to wrap each function? Can't we take the pointer, deref it once then call all the callbacks?

Reply:
objection

Reply:
sustained